components/event-publisher-proxy/pkg/legacy-events/legacy.go

```diff
@@ -123,6 +125,9 @@ func (t Transformer) TransformLegacyRequestsToCE(writer http.ResponseWriter, req
 		return nil
 	}
 
+	// Add tracing context to cloud events
+	tracing.AddTracingContextToCEExtensions(request.Header, event)
```

Comment:
Since this is for legacy-events, Is it also handled for cloudevents?

Reply:
No, it's not. Fixed it.